auslib/web/admin/swagger/api.yaml

```diff
@@ -1893,90 +1906,970 @@ paths:
+        - text/html
+      externalDocs:
+        url: "http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#scheduled-changes-object-sc-id-signoffs"
+        description: Removes an existing Signoff from the Scheduled Change for the Permission Required Signoff.
+      parameters:
+        - $ref: '#/parameters/sc_id_param'
+        - $ref: '#/parameters/csrfTokenParam'
+      responses:
+        '200':
+          description: Successfully deleted the signoff on the given scheduled change.
+        '404':
+          $ref: '#/responses/resourceNotFound'
+
+definitions:
+  4xxClientErrorResponseModel:
+    title: "4xx Response"
+    description: "Invalid request's response object. The 4xx class of status codes is intended for situations in which the client seems to have erred. This model describes the response format"
+    type: object
+    # currently we're using 'detail' and 'status' in the UI.
+    # 'exception' is seldom used. Therefore rest of fields aren't always required.
+    required:
+      - detail
+      - status
+      - title
+    properties:
+      detail:
+        description: An human readable explanation specific to this occurrence of the problem.
+        type: string
+        example: "field_name is a required property"
+      exception:
+        description: An human readable explanation specific to this occurrence of the problem. Same as detail. Used in some UI forms.
+        type: string
+        example: "field_name is missing"
+      status:
+        description: The HTTP status code generated by the origin server for this occurrence of the problem.
+        type: integer
+        format: int32
+        minimum: 400
+        maximum: 499
+        example: 400
+      title:
+        description: A short, human-readable summary of the problem type. It SHOULD NOT change from occurrence to occurrence of the problem, except for purposes of localisation.
+        type: string
+        example: "Bad Request"
+      type:
+        description: "An absolute URI that identifies the problem type.  When dereferenced, it SHOULD provide human-readable documentation for the problem type (e.g., using HTML).When this member is not present its value is assumed to be 'about:blank'"
+        type: string
+        example: "about:blank"
+
+  CSRFModel:
+    title: CSRF Token
+    description: CSRF Token in POST/PUT or UPDATE requests
+    type: object
+    required:
+      - csrf_token
+    properties:
+      csrf_token:
+        type: string
+        example: "1491342563##c4e6fef0b978e6c89af9ff1015e67b9ca7c45d14"
+
+  CountModel:
+    title: Count
+    description: count of total objects/elements fetched
+    type: object
+    required:
+      - count
+    properties:
+      count:
+        description: count of total values fetched from DB
+        type: integer
+        minimum: 0
+        format: int32
+        example: 1
+
+  DataVersionModel:
+    title: Data Version
+    description: "model to re-use 'data_version' field."
```

Comment:
This model is used multiple API operations. Those API request/response json bodies that have 'required' set to true explicitly define in their own 'required' list in API's definition sections, ([example](https://github.com/aksareen/balrog/blob/ddee69f55db14de816c84d3615426661c3dce846/auslib/web/admin/swagger/api.yaml#L759), while some APIs have it as optional , hence not always required. [Example](https://github.com/aksareen/balrog/blob/ddee69f55db14de816c84d3615426661c3dce846/auslib/web/admin/swagger/api.yaml#L300), [example](https://github.com/aksareen/balrog/blob/ddee69f55db14de816c84d3615426661c3dce846/auslib/web/admin/swagger/api.yaml#L3132)